localisation/en.json

```diff
@@ -216,9 +216,9 @@
   "sysstats.error": "I was unable to request statistics information from the Systems API due to an error.",
   
   "rescue.pwreminder": "Hi %{nick}, you seem to have some unfiled paperwork; %{rescues}.",
-  "rescue.pwreminder.meme": "%{nick}, you didn't file your paperwork last night; %{rescues}. I am watching you, %{nick}.",
+  "rescue.pwreminder.meme": "%{nick}, you didn't file your paperwork last night; %{rescues}. I believe in you, %{nick}! :)",
```

Comment:
I am willing to change the rescue.pwreminder.special one, but rescue.pwreminder.meme is a Monster's Inc reference and I'd like to keep it :P